### PR TITLE
Hutlar Dungeon redux

### PR DESCRIPTION
**Pull request by ZarquonZ**
Cleaned it up, did the fixes mentioned in the comments, and added the pitfall trap trigger as a .utt.

### CODE REVIEW

File: SWLOR.Game.Server/Feature/TrapDefinition/QionSlugDeath.cs [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+        /// <summary>
+        /// When the creatures spawn, this will broadcasts an environmental message describing the narrative circumstances of that spawn in.
+        /// It has to be ChatChannel.DMTalk - it won't work if it's ChatChannel.PlayerTalk.
+        /// </summary>
+        [NWNEventHandler("crea_spawn_aft")]
+        public static void MessageOnDeath()
+        {
+            if (GetTag(OBJECT_SELF) != "qion_hive_larvae")
+            return;
```
Comment (zunath) [inline SWLOR.Game.Server/Feature/TrapDefinition/QionSlugDeath.cs:39]: Tab this over please.

File: SWLOR.Game.Server/Feature/TrapDefinition/QionSlugDeath.cs [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+            uint sender = OBJECT_SELF;
```
Comment (zunath) [inline SWLOR.Game.Server/Feature/TrapDefinition/QionSlugDeath.cs:41]: uint -> var

Alternatively just use OBJECT_SELF in your Messaging.SendMessageNearbyToPlayers call below.

File: SWLOR.Game.Server/Feature/TrapDefinition/QionSlugDeath.cs (hunk 1 above)
Comment (ZarquonZ) [reply SWLOR.Game.Server/Feature/TrapDefinition/QionSlugDeath.cs]: Amended!

File: SWLOR.Game.Server/Feature/TrapDefinition/QionSlugDeath.cs (hunk 2 above)
Comment (ZarquonZ) [reply SWLOR.Game.Server/Feature/TrapDefinition/QionSlugDeath.cs]: Amended!

**Review by zunath — APPROVED** [review]

**Pull request merged**